package.json

```diff
@@ -26,6 +26,7 @@
   },
   "dependencies": {
     "@bcgov/bc-sans": "^2.1.0",
+    "@chakra-ui/icons": "^2.1.1",
```

Comment:
Karen mentioned the gov uses icons from Font Awesome 6  Regular so we should add that in and import icons from there